docs/plugin.md

```diff
@@ -97,20 +97,48 @@ createPlugin(document.getElementById('app')!); // app is a <div> element
 To use the plugin (with the React UI) inside another React app:
 
 ```ts
-function MolStarWrapper() {
-    const parent = React.createRef<HTMLDivElement | null>();
-
-    useEffect(() => {
-        let plugin: PluginContext | undefined = undefined;
-        async function init() {
-            plugin = createPlugin(parent.current);
-        }
-        init();
-        return () => { plugin?.dispose(); };
-    }, []);
-
-    return <div ref={parent} style={{ width: 640, height: 480 }} />;
+import { useEffect, createRef } from "react";
+import { createPluginUI } from "molstar/lib/mol-plugin-ui";
+import { PluginUIContext } from "molstar/lib/mol-plugin-ui/context";
+import "molstar/build/viewer/molstar.css";
+
+declare global {
+  interface Window {
+    molstar: PluginUIContext;
+  }
 }
+
+window.molstar = window.molstar || {};
+
+export function MolStarWrapper() {
+  const parent = createRef<HTMLDivElement>();
+
+  useEffect(() => {
+    async function init() {
+      window.molstar = await createPluginUI(parent.current as HTMLDivElement);
+
+      if (window.molstar) {
```

Comment:
This condition doesn't do anything since you do the create right above it.

Reply:
I wanted to avoid trying these steps if the creation failed, maybe use a `try/catch` instead?

Reply:
If it fails, it will throw anyway and this line will never be hit.

Reply:
done in https://github.com/ptourlas/docs/commit/49a5bda6736f4d40d8da4f368977553d3bdc5093